helix-query.yaml

```diff
@@ -74,14 +74,10 @@ indices:
         value: |
           match(attribute(el, 'content'), 'https:\/\/[^/]+(\/.*)')
       description:
-        select: head > meta[name="og:description"]
+        select: head > meta[property="og:description"]
         value: |
           attribute(el, 'content')
       category:
         select: head > meta[name="category"]
         value: |
-          attribute(el, 'content')
-      date:
```

Comment:
@hsaginor  Can we remove the date from the product index ? It looks like it's missing in the documents but I'm not sure if it's missing because it doesn't exist or because the importer missed it.

Reply:
@icaraps Not seeing it on original product pages and the importer is not pulling it. We can remove it if you don't need it. I guess the question is what fields do we need? Just keep or add what we need.